PayloadPkg/OsLoader/LoadImage.c

```diff
@@ -299,7 +311,9 @@ GetBootImageFromFs (
     goto Done;
   }
 
-  if (LoadedImage->LoadImageType == LoadImageTypeExtra0) {
+  // TODO: RTCM compatibility
+  // if (LoadedImage->LoadImageType == LoadImageTypeExtra0) {
+  if (0) {
```

Comment:
TCC binaries (including RTCM) is dropped from ADL/RPL. To keep it works for old platforms, let's keep it until TCC is removed from all platforms. It looks all the RTCM file name is "/boot/sbl_rtcm", maybe we could check if file name is rtcm and extra0, we could keep it as-is.

Reply:
Fixed